packages/native-stack/src/types.tsx

```diff
@@ -452,6 +454,43 @@ export type NativeStackNavigationOptions = {
    * @platform ios
    */
   gestureResponseDistance?: ScreenProps['gestureResponseDistance'];
+  /**
+   * Changes the gesture for dismissing the screen to the previous one.
+   *
+   * Supported values:
+   * - "swipeRight": swipe right to dismiss the screen
+   * - "swipeLeft": swipe left to dismiss the screen
+   * - "swipeUp": swipe up to dismiss the screen
+   * - "swipeDown": swipe down to dismiss the screen
+   * - "verticalSwipe": swipe up or down to dismiss the screen
+   * - "horizontalSwipe": swipe left or right to dismiss the screen
+   * - "twoDimensionalSwipe": swipe in any direction to dismiss the screen
+   *
+   * Only supported on iOS and Android.
+   */
+  goBackGesture?: GoBackGesture;
+  /**
+   * Changes the animation for dismissing the screen when making a swipe gesture.
+   * You can choose one of the built-in transition presets, by using `ScreenTransition` from `react-native-reanimated`.
```

Comment:
Probably should re-export these from `/native-stack`. In the future if we implement transition animations on the Web, we can make the same code work on the Web. And generally cleaner not to leak the abstraction.

Reply:
It may be problematic for now, since we're only [exporting](https://github.com/software-mansion/react-native-screens/blob/546e72f56359eacade1eb35700dcb19aa8434f66/src/native-stack/index.tsx#L23) some of the types in `native-stack` package, so changing this export could be a blocker, potentially.
Also, I wouldn't re-export this type from reanimated (obviously we'd make it a strict dependency).